error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/Slf4jLogDeclarationTest.java

```diff
@@ -0,0 +1,98 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode;
+import org.junit.jupiter.api.Test;
+
+final class Slf4jLogDeclarationTest {
+  @Test
+  void replacement() {
+    BugCheckerRefactoringTestHelper.newInstance(Slf4jLogDeclaration.class, getClass())
```

Comment:
Usually we have the `identification` test which is the most extensive one where we test all the edge cases. The replacement test only shows that the replacement logic is correct. Apart from that we don't want to have all the different cases in the `replacement` test.

Reply:
Thanks @rickie .

Updating the test actually made me spot a fault in the implementation logic. I updated the code in the checker and the tests. PTAL 🙏